### DISCUSSION TITLE
Help on Smoothing I80 dataset

### DISCUSSION THREAD

**Discussion by NachuanLi** [Rim-El-Ballouli/NGSIM-US-101-trajectory-dataset-smoothing#3] [Q&A]
Hi  Dr Rim El Ballouli,
I wonder if you have available i80 dataset that is smoothed (as unknow type object error is keeping showing up when we are doing smoothing on i80 dataset on the three time windows), or raw data set of US101, partitioned by three 15min time windows. These will be very useful for our research. Thanks a lot
Nachuan Li

**Comment by Rim-El-Ballouli** [comment]
Hello Nachuanli,

Thank you for your interest in this repo :). To answer shortly your question, I do not have a smoothed version of the i80 dataset. However maybe the current code could be adapted for the i80, but first I have few question to clarify.

1. Can you share with me the description of columns of the i80 dataset, so I can see if an adaptation is possible?
2. Can you clarify what you by "partioning by three 15 min time window"?

Rim

**Comment by NachuanLi** [comment]
Hi Dr Rim El Ballouli,

Thanks a lot for replying. Below are the columns of my i80 dataset. I would also attach the error message below, and I hope you could figure out what is wrong based on the information.

Thanks a lot,
Nachuan Li

['Vehicle_ID', 'Frame_ID', 'Total_Frames', 'Global_Time', 'Local_X',        'Local_Y', 'Global_X', 'Global_Y', 'v_length', 'v_Width', 'v_Class',        'v_Vel', 'v_Acc', 'Lane_ID', 'O_Zone', 'D_Zone', 'Int_ID', 'Section_ID',        'Direction', 'Movement', 'Preceding', 'Following', 'Space_Headway',        'Time_Headway', 'Location'].
[cid:271394c7-afe8-49a5-8e50-d0f2bd8a0438]
[cid:827e1041-e6a2-4da0-834f-791d611c03ae]

________________________________

**Reply by Rim-El-Ballouli** [reply]
- I think you forgot to attach the error message?
- Also, which window size are you using? 3 or 15?

**Comment by NachuanLi** [comment]
Hi Dr Rim El-Ballouli,
I used the default value of window size, which is 3. I apologize that you did not see the attached error message, and I will re-attach here.
Thanks,
Nachuan Li
[cid:a8b54918-4c2a-4550-993f-d742d0b086a0]
[cid:e49a6e05-7d55-4135-afee-7661fbb47a8b]


________________________________

**Reply by Rim-El-Ballouli** [reply]
Hi again,  I still can't access the images. I only see the CID reference of the images. I cant seem to figure out how to open them. Is there a special way to open that? Also, cant you just embed the image directly? I do not have a problem running the code on i80 with window 3.  Therefore, I will need to see the errors you are having to help. 
Thanks!

**Comment by NachuanLi** [comment]
Hi Dr Rim-El-Ballouli,
Here are the images of error messages with png format, I hope that you could open them.
Thanks,
Nachuan Li

________________________________
